tutorial.md

```diff
@@ -0,0 +1,378 @@
+# Rpi3 OS on Rust
+
+## tl;dr
```

Comment:
This line is visible

Reply:
It is OK to use it, https://en.wikipedia.org/wiki/Wikipedia:Too_long;_didn%27t_read